linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_files_permissions/rule.yml

```diff
@@ -1,24 +1,18 @@
-{{%- if product in ["debian10", "debian11", "ubuntu1604", "ubuntu1804", "ubuntu2004", "ubuntu2204", "sle15", "sle12"] %}}
-    {{%- set rsyslog_perm='640' %}}
-{{%- else %}}
-    {{%- set rsyslog_perm='600' %}}
-{{%- endif %}}
-
 documentation_complete: true
 
 title: 'Ensure System Log Files Have Correct Permissions'
 
 description: |-
     The file permissions for all log files written by <tt>rsyslog</tt> should
-    be set to {{{ rsyslog_perm }}}, or more restrictive. These log files are determined by the
+    be set to 640, or more restrictive. These log files are determined by the
     second part of each Rule line in <tt>/etc/rsyslog.conf</tt> and typically
     all appear in <tt>/var/log</tt>. For each log file <i>LOGFILE</i>
     referenced in <tt>/etc/rsyslog.conf</tt>, run the following command to
     inspect the file's permissions:
     <pre>$ ls -l <i>LOGFILE</i></pre>
-    If the permissions are not {{{ rsyslog_perm }}} or more restrictive, run the following
+    If the permissions are not 640 or more restrictive, run the following
     command to correct this:
-    <pre>$ sudo chmod {{{ rsyslog_perm }}} <i>LOGFILE</i></pre>"
+    <pre>$ sudo chmod 640 <i>LOGFILE</i></pre>"
```

Comment:
Just a small comment, since stricter permissions are allowed, it would be nice to use a command which only removes the excessive permission instead of setting `640` precisely. But I know this is not in the intention of this PR. : )